apps/qubit/modules/term/templates/indexSuccess.php

```diff
@@ -254,11 +254,12 @@
           <?php endif; ?>
 
         <div id="sort-header">
-          <?php echo get_partial('default/sortPicker',
-            array(
-              'options' => array(
-                'lastUpdated' => __('Most recent'),
-                'alphabetic' => __('Alphabetic')))) ?>
+          <?php echo get_partial('default/sortPicker', array(
+            'options' => array(
+              'lastUpdated' => __('Most recent'),
+              'alphabetic'  => __('Alphabetic'),
+              'identifier'  => __('Reference code'),
+              'date'        => __('Date')))) ?>
```

Comment:
You need to update [this](https://github.com/artefactual/atom/blob/a93ba709ba3b65e781c02bd5af1fff1c529e5095/apps/qubit/modules/term/actions/indexAction.class.php#L276-L288) too, right?